cookbooks/aws-parallelcluster-platform/resources/arm_pl/arm_pl_ubuntu22.rb

```diff
@@ -13,14 +13,14 @@
 # See the License for the specific language governing permissions and limitations under the License.
 
 provides :arm_pl, platform: 'ubuntu' do |node|
-  node['platform_version'].to_i == 22
+  node['platform_version'].to_i >= 22
 end
 
 use 'partial/_arm_pl_common.rb'
 
 action_class do
   def armpl_platform
-    "Ubuntu-#{node['platform_version']}"
+    "Ubuntu-22.04"
```

Comment:
Nitpicks:
Rename the file to *22+.rb` and add a comment that armpl_plaform is hardcoded to `Ubuntu-22.04` as that the folder in s3 which we look for the tar file